packages/datagrid/src/datagrid.ts

```diff
@@ -3298,6 +3375,20 @@ class DataGrid extends Widget {
     this._canvasGC.stroke();
   }
 
+  /**
+   * Emit the `cellClick` signal for the data grid.
+   */
+  protected emitCellClick(args: DataGrid.ICellClick): void {
```

Comment:
Maybe make this a `protected` class member instead of having a separate method?

Reply:
Updated!